cmd/deploy/wait.go

```diff
@@ -72,7 +72,9 @@ func (dc *DeployCommand) waitForResourcesToBeRunning(ctx context.Context, opts *
 				return err
 			}
 			areAllRunning := true
-			for _, d := range dList {
+			ds := make([]v1.Deployment, len(dList))
+			for i := range ds {
```

Comment:
Why is this change? We are discarding what's in `dList`, and looping over an empty slice

Reply:
my bad, this was a fix from a deepsource detection. fixed now